data/json/items/gunmod/magazine.json

```diff
@@ -12,13 +12,15 @@
     "symbol": ":",
     "color": "green",
     "location": "magazine",
-    "mod_targets": [ "rifle" ],
-    "acceptable_ammo": [ "223", "308" ],
+    "mod_targets": [ "rifle", "launcher" ],
+    "acceptable_ammo": [ "223", "308", "50", "40mm" ],
```

Comment:
This will allow single shot break-action grenade launchers to use belts. That entirely removes the advantage the larger launchers previously had and is also an impossible modification IRL